src/main/java/net/sf/jabref/logic/msbib/BibTeXConverter.java

```diff
@@ -77,12 +81,26 @@ public static BibEntry convert(MSBibEntry entry) {
             fieldValues.put(MSBIB_PREFIX + "accessed", entry.dateAccessed);
         }
 
+        if(entry.journalName != null)
```

Comment:
Don't forget to add the space after the `if` and move the bracket in the same line.